### DISCUSSION TITLE
Diffusion Curves

### DISCUSSION THREAD

**Discussion by abu-irrational** [blend2d/blend2d#100] [Ideas]
I imagine it won't be easy, but what do you think about this new kind of gradient ?

https://en.wikipedia.org/wiki/Diffusion_curve

On the right-side a bunch of curves with diffusion colors defined on them,
On the left-side the resulting raster images

<img width="579" alt="image" src="https://user-images.githubusercontent.com/13416847/105093538-2f05a400-5aa3-11eb-80a1-64274f0b7149.png">

**Comment by Wiladams** [comment]
looks like a cross between isosurface and interpolation.

**Comment by Sosukodo** [comment]
That's very cool.. How was (e) created? by hand?
